src/otx/algo/visual_prompting/zero_shot_segment_anything.py

```diff
@@ -124,14 +129,19 @@ def _point_selection(
         self,
         mask_sim: Tensor,
         ori_shape: Tensor,
-        threshold: Tensor,
-        num_bg_points: Tensor,
+        threshold: Tensor | None = None,
+        num_bg_points: Tensor | None = None,
     ) -> tuple[Tensor, Tensor]:
         """Select point used as point prompts."""
+        if threshold is None:
+            threshold = torch.tensor([0.0], dtype=torch.float32)
+        if num_bg_points is None:
+            num_bg_points = torch.tensor(1, dtype=torch.int64)
+
         _, w_sim = mask_sim.shape
 
         # Top-last point selection
-        bg_indices = mask_sim.flatten().topk(num_bg_points[0, 0], largest=False)[1]
+        bg_indices = mask_sim.flatten().topk(num_bg_points.item(), largest=False)[1]
```

Comment:
If `num_bg_points` is not a CPU tensor, calling `.item()` makes it to be transferred from accelerator to main memory and the thread will be being blocked to wait for it. Is this change necessary?

Reply:
Updated `num_bg_points` to `int` at https://github.com/openvinotoolkit/training_extensions/pull/2955/commits/f1aa67c97de4ce745533c2b3745d2e6770974c81.